src/cdk/testing/tests/testbed.spec.ts

```diff
@@ -12,6 +12,7 @@ import {MainComponentHarness} from './harnesses/main-component-harness';
 import {SubComponentHarness, SubComponentSpecialHarness} from './harnesses/sub-component-harness';
 import {TestComponentsModule} from './test-components-module';
 import {TestMainComponent} from './test-main-component';
+import {querySelectorAll as piercingQuerySelectorAll} from 'kagekiri';
```

Comment:
Would a querySelector that cuts through the shadows be a `luminousQuerySelectorAll`?
<sub>_that's a joke_</sub>